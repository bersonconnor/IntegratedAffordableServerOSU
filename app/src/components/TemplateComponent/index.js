import React from "react";
import PropTypes from "prop-types";
import "./scss/template-component.scss";

const TemplateComponent = ({ name }) => {
  return <div>This is a {name}</div>;
};

TemplateComponent.propTypes = {
  /*
   *  This is an example comment for a prop
   */
  name: PropTypes.string
};

TemplateComponent.defaultProps = {
  name: "TemplateComponent"
};

export default TemplateComponent;
