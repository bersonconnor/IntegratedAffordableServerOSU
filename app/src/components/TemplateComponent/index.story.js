import React from "react";
import { storiesOf } from "@storybook/react";
import TemplateComponent from "./";

storiesOf("TemplateComponent", module).add("default", () => (
  <TemplateComponent />
));
