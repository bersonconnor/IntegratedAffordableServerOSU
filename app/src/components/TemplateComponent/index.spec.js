import React from "react";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";
import TemplateComponent from "./";

describe("TemplateComponent", () => {
  describe("Snapshots", () => {
    it("should match snapshot for default TemplateComponent", () => {
      expect(renderer.create(<TemplateComponent />).toJSON()).toMatchSnapshot();
    });
  });
});
