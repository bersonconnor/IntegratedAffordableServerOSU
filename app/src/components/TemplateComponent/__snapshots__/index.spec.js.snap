// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TemplateComponent Snapshots should match snapshot for default TemplateComponent 1`] = `
<div>
  This is a 
  TemplateComponent
</div>
`;
